cmd/utils/flags.go

```diff
@@ -520,8 +520,8 @@ var (
 	// Vault unlock settings
 	VaultAddressFlag = cli.StringFlag{
 		Name:  "vault.address",
-		Usage: "Vault server address",
-		Value: "http://127.0.0.1:8200",
+		Usage: "Vault server address (like http://127.0.0.1:8200)",
```

Comment:
typo? `e.g.`

Reply:
do it [f22d6b8](https://github.com/antimatter-dao/bas-template-bsc/pull/23/commits/f22d6b8b20800b6cab523083ea8e9e04f191c341)